### PR TITLE
Release Nistats 0.0.1b1 prep

### PR DESCRIPTION
**Pull request by kchawla-pi**
Updated list of contributors & version number for release

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -28,6 +28,21 @@ Fixes
 Contributors
 ------------
 
+The following people contributed to this release (in alphabetical order)
+
+	Ana Luisa Pinho
+	Anthony Gifuni
+	Bertrand Thirion
+	Chris Markiewicz
```
Comment (effigies) [inline doc/whats_new.rst]: Duplicate

```suggestion
```

Comment (kchawla-pi) [reply doc/whats_new.rst]: What's your commit email?

Comment (kchawla-pi) [reply doc/whats_new.rst]: Think I found it. Tha stanford one.

Comment (effigies) [reply doc/whats_new.rst]: Yeah, mostly Stanford. Some GitHub-generated commits will show up on gmail. These two lines will generally work:

https://github.com/nipy/nipype/blob/536c57da150d157dcb5c121af43aaeab71cdbd5f/.mailmap#L34-L35

**Pull request merged**